specification/monitor/data-plane/readme.md

```diff
@@ -0,0 +1,147 @@
+# MonitorClient
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for MonitorClient.
+
+
+
+---
+## Getting Started
+To build the SDK for MonitorClient, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+---
+
+## Configuration
+
+
+
+### Basic Information
+These are the global settings for the MonitorClient API.
+
+``` yaml
+title: MonitorClient
+description: Monitor Management Client
+openapi-type: arm
+
+tag: package-2018-09
+
+```
+### Tag: package-2018-09
+
+These settings apply only when `--tag=package-2018-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-09'
+input-file:
+- data-plane/preview/2018-09-01-preview/azureMonitorCustomMetricsIngestionApi.json
```

Comment:
This path is relative to the location of the README.md file, it should be `preview/2018-09-01-preview/azureMonitorCustomMetricsIngestionApi.json`.

Reply:
My bad. Thanks for catching it. Fixed